mne/preprocessing/ica.py

```diff
@@ -184,6 +184,10 @@ class ICA(ContainsMixin):
         The measurement info copied from the object fitted.
     `n_samples_` : int
         the number of samples used on fit.
+    `labels_` : dict
+        A dictionary of independent component indices, grouped by kinds
+        of artifacts. This attribute is set by some of the artifact detection
```

Comment:
Mustn't necessarily be artifacts - in theory, you can also identify e.g. central-midline ICs this way, although AFAIK at least CORRMAP has not been used for that purpose so far. Maybe ' A dictionary of independent component indices, grouped by IC type. This attribute is set by some of the artifact detection'.